paris-processor/src/main/java/com/airbnb/paris/processor/writers/StyleApplierJavaClass.kt

```diff
@@ -57,12 +57,12 @@ internal class StyleApplierJavaClass(
             }
         }
 
-        if (!styleableInfo.styleableResourceName.isEmpty()) {
+        if (styleableInfo.styleableResourceName.isNotEmpty()) {
             method("attributes") {
                 override()
                 protected()
                 returns(ArrayTypeName.of(Integer.TYPE))
-                addStatement("return \$T.styleable.\$L", RElement, styleableInfo.styleableResourceName)
+                addStatement("return \$T.styleable.\$L", styleableInfo.styleableRClassName ?: RElement, styleableInfo.styleableResourceName)
```

Comment:
When would `styleableInfo.styleableRClassName` be `null` in this conditional?

Reply:
This is the case where a class has no `attrs`/`styleablechild`, it seems that this code doesn't need to be generated, but to preserve the existing behavior we would need look at the current module's `R` file.

This could also an invalid cause if we were to change the no attr/styleablechild warning to an error.